src/Avalonia.Base/Controls/IInternalScroller.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Avalonia.Controls.Primitives;
+
+internal interface IInternalScroller
```

Comment:
Maybe add a comment here explaining why it's needed with a `TODO12` comment explaining how we may want to refactor this when we can make breaking changes.

Reply:
+